### ISSUE TITLE
Fdelete() failed on u:\host

### ISSUE THREAD

**Issue by xdelatour** [freemint/freemint#303]
Tested with latest `freemint-1-19-5e9fe146-040-aranym.zip`

```
Fdelete("c:\\clipbrd\scrap.txt") returns 0 (OK)
```

```
Fdelete("u:\\host\\clipbrd\scrap.txt") returns -36 (EACCDN)
```

On the one hand qed is able to update u:\host\clipbrd\scrap.txt, but on the other hand Teradesk can't delete the file: 'Can not delete scrap.txt: Access denied. Item is protected or it already exists.'

Do the files stay open in u:\host for some reason?

**Comment by mikrosk** [comment]
That looks more like an Aranym issue to me. I'd imagine that Aranym keeps that file open all the time for exactly that - realtime update from the Atari side.

But that's just my guess of course.

**Comment by xdelatour** [comment]
Indeed, required by aranym.xfs, scrap.txt cannot not be deleted https://github.com/freemint/freemint/blob/c022b02d3aee926afdb5af2600deda0c54949e23/sys/xfs/aranym/arafs.c#L145-L147

This means that if we want to implement scrp_clear() we have to check for NativeFeatures and truncate the file
